spec/controllers/exercises_controller_spec.rb

```diff
@@ -12,11 +12,18 @@
       get :show, id: exercise.id
     end
 
-    it "changes exercise status to finished" do
-      patch :mark_as_finished, exercise_id: exercise.id
+    it "changes exercise status to finished given a valid answer" do
```

Comment:
"changes the exercises status to finished when a valid answer is given" sounds a bit better